jennchenn/uwexplore
# Sign Up/Login Components and Validation

## Implementation description

- signup modal
- login modal



## Steps to test

1. open login/sign up
2. toggle through both the modals
3. styling present
4. inputs working
5. test out email regex
6. test out password regex
7. make sure error messages present
